src/tests/functional/plugin/cpu/single_layer_tests/lstm_sequence.cpp

```diff
@@ -43,97 +71,82 @@ class LSTMSequenceCPUTest : public testing::WithParamInterface<LSTMSequenceCpuSp
-        size_t hidden_size;
-        size_t input_size;
+        std::vector<InputShape> inputShapes;
+        ngraph::helpers::SequenceTestsMode seqMode;
         std::vector<std::string> activations;
-        std::vector<float> activations_alpha;
-        std::vector<float> activations_beta;
         float clip;
-        ngraph::op::RecurrentSequenceDirection direction;
-        InferenceEngine::Precision netPrecision;
+        ov::op::RecurrentSequenceDirection direction;
+        ElementType netPrecision;
+        CPUSpecificParams cpuParams;
+        std::map<std::string, std::string> additionalConfig;
 
-        std::tie(basicParamsSet, cpuParams, additionalConfig) = this->GetParam();
+        std::tie(inputShapes, seqMode, activations, clip, direction, netPrecision, cpuParams, additionalConfig) = this->GetParam();
         std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
-        std::tie(m_mode, seq_lengths, batch, hidden_size, input_size, activations, clip, direction, netPrecision, targetDevice) = basicParamsSet;
-
-        size_t num_directions = direction == ngraph::op::RecurrentSequenceDirection::BIDIRECTIONAL ? 2 : 1;
-        m_max_seq_len = seq_lengths;
-        std::vector<std::vector<size_t>> inputShapes = {
-            {{batch, seq_lengths, input_size},
-             {batch, num_directions, hidden_size},
-             {batch, num_directions, hidden_size},
-             {batch},
-             {num_directions, 4 * hidden_size, input_size},
-             {num_directions, 4 * hidden_size, hidden_size},
-             {num_directions, 4 * hidden_size}},
-        };
+        targetDevice = CommonTestUtils::DEVICE_CPU;
+
+        init_input_shapes(inputShapes);
+
+        const size_t inputSize = inputDynamicShapes.front()[2].get_length();
+        const size_t hiddenSize = inputDynamicShapes[1][2].get_length();
+        const size_t numDirections = direction == ov::op::RecurrentSequenceDirection::BIDIRECTIONAL ? 2 : 1;
 
         // method MKLDNNMemoryDesc::isSame can't correct compute layout for tensor with strides = 1
         // returned output format always tnc
         if (inFmts.size() >= 3) {
             for (size_t i = 1; i < 3; i++) {
-                if (ngraph::shape_size(inputShapes[i]) == 1) {
+                if (inputDynamicShapes[i].is_static() && ov::shape_size(inputDynamicShapes[i].to_shape()) == 1) {
                     inFmts[i] = tnc;
                 }
             }
         }
 
         configuration.insert(additionalConfig.begin(), additionalConfig.end());
 
-        if (additionalConfig[PluginConfigParams::KEY_ENFORCE_BF16] == PluginConfigParams::YES) {
-            inPrc = outPrc = Precision::BF16;
-        } else {
-            inPrc = outPrc = netPrecision;
-        }
-
-        selectedType += "_";
-        selectedType += outPrc.name();
+//        if (additionalConfig[InferenceEngine::PluginConfigParams::KEY_ENFORCE_BF16] == InferenceEngine::PluginConfigParams::YES) {
```

Comment:
uncomment please

Reply:
That does not work properly with new tests API.